pages/ksphere/konvoy/1.4/release-notes/open-source-attribution/index.md

```diff
@@ -114,14 +114,3 @@ The table below lists the third party open source software which is provided by
 |utils                      |v0.0.0-20191010214722-8d271d903fe4  |[Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)                                                 |
 |controller-runtime         |v0.3.1-0.20191022174215-ad57a976ffa1|[Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)                                                 |
 |yaml                       |v1.1.0                              |[MIT License](https://github.com/weapp/boto/blob/master/LICENSE.txt)                                              |
-
-
-## Software Sources
-
-The table below lists the Sources for the Free Software under GNU General Public License, and Mozilla Public License
```

Comment:
@samvantran or @dkoshkin I think these are the things Senthil added a while back. Why do we want to take them back out? Was this a backport issue?